src/sync/installation.ts

```diff
@@ -128,13 +129,15 @@ export const updateJobStatus = async (
 		logger.info("Organization has been deleted. Other active syncs will continue.");
 		return;
 	}
+	const { edges } = taskPayload;
+	const isComplete = !edges?.length || taskPayload.isDone;
```

Comment:
I see this changes the logic (`isDone` wasn't there before) and there's neither a test nor a feature-flag...

Reply:
Actually it is not used at all, because edges is undefined in this case...

Reply:
So I guess that's OK :)